plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -306,5 +306,5 @@ def _move_file(self, unit):
         :param unit: a pulp unit
         :type  unit: pulp.plugins.model.Unit
         """
-        _logger.debug('moving files in to place for Unit {}'.format(unit))
```

Comment:
`'{}'.format` may not work, but reverting to percent interpolation is a bit harsh. how about `'{0}'.format`?